src/gimelstudio/core/project_file.py

```diff
@@ -86,52 +86,56 @@ def SaveNodesData(self, nodes):
                                     location="POSITION")
 
-            # Create the node properties
-            prop_data = node_data["properties"]
-            for prop in prop_data:
-                node.NodeEditProp(idname=prop_data[prop]["idname"],
+                # Create the node properties
+                prop_data = node_data["properties"]
+                for prop in prop_data:
+                    node.NodeEditProp(idname=prop_data[prop]["idname"],
                                   value=prop_data[prop]["value"], 
                                   render=False)
 
-        # Create the parameters and node connections
-        for node_id in nodes_data:
-            node_data = nodes_data[node_id]
-            node_objs = nodegraph.GetNodes()
-            param_data = node_data["parameters"]
-            node = node_objs[node_id]
-
-            for socket in node.GetSockets():
-                if socket.direction == SOCKET_INPUT:
-                    dst_socket = socket
-
-                    # Find the output socket, connect the input socket
-                    src_node_id = param_data[dst_socket.idname]["binding"]  
-                    if src_node_id != "":  # Make sure there actually is a node connected
-                        src_node = node_objs[src_node_id]
-
-                        for socket in src_node.GetSockets():
-                            # We're assuming there is only one output
-                            if socket.direction == SOCKET_OUTPUT:
-                                src_socket = socket
-
-                        # Finally, connect the sockets
-                        nodegraph.ConnectNodes(src_socket, dst_socket)
+            # Create the parameters and node connections
+            for node_id in nodes_data:
+                node_data = nodes_data[node_id]
+                node_objs = nodegraph.GetNodes()
+                param_data = node_data["parameters"]
+                node = node_objs[node_id]
+
+                for socket in node.GetSockets():
+                    if socket.direction == SOCKET_INPUT:
+                        dst_socket = socket
+
+                        # Find the output socket, connect the input socket
+                        src_node_id = param_data[dst_socket.idname]["binding"]  
+                        if src_node_id != "":  # Make sure there actually is a node connected
+                            src_node = node_objs[src_node_id]
+
+                            for socket in src_node.GetSockets():
+                                # We're assuming there is only one output
+                                if socket.direction == SOCKET_OUTPUT:
+                                    src_socket = socket
+
+                            # Finally, connect the sockets
+                            nodegraph.ConnectNodes(src_socket, dst_socket)
             
+        except:
+            wx.MessageBox(message="Unable to create graph from file, is the file corrupted?")
```

Comment:
This string should be translatable which can be done by doing the following.
```suggestion
            wx.MessageBox(message=_("Unable to create graph from file, is the file corrupted?"))
```

The other part is the message string itself. I'd suggest using `Unable to load node graph from file. The file may be corrupt.` instead of `Unable to create graph from file, is the file corrupted?`.

@Correct-Syntax Thoughts on the message string?

Reply:
I agree with your suggested change of wording here, @iwoithe